docs/tutorial.rst

```diff
@@ -388,4 +388,48 @@ Let's do the bootstrap test on the two categories.
 
 Drawing Maps
 ------------
-To come.
+The main class in the maps module is the Map class. In this code we create a default map. Maps can be displayed or converted to html.
```

Comment:
"In this code we create a default map" -> "In this code we create a default Map" (to distinguish against Python maps)